scripts/lighthouse/lighthouse.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable eslint-comments/no-unlimited-disable */
+/* eslint-disable */
+const lighthouse = require('lighthouse');
+const chromeLauncher = require('chrome-launcher');
+const table = require('markdown-table');
+
+const ROUTES_MAPPINGS = require('./pages');
+const checkLighthouseScore = require('./lh-check-scores');
+const config = require('./lh-config');
+
+const toMarkdown = (results = {}) =>
+  table([['Metric', 'Value']].concat(Object.entries(results)));
+
+const generateOutput = (scores, url) => {
+  console.log('URL', url);
+
+  if (scores.successes) {
+    console.log('\n===== LIGHTHOUSE PASS! =====\n');
```

Comment:
Can we have some colour in here? You can use the `colors` dependency that's already used elsewhere.
https://www.npmjs.com/package/colors

Reply:
Sure thing!